lib.nix

```
@@ -0,0 +1,47 @@
+{ lib, fetchgit, fetchgitPrivate, gitignoreSource, fetchFromGitHub }:
```

This way callers don't have to grab this themselves.
```suggestion
{ lib
, fetchgit
, fetchgitPrivate
, gitignoreSource ? (import ./dep/gitignore.nix { inherit (pkgs) lib; }).gitignoreSource
, fetchFromGitHub
}:
```